### ISSUE TITLE
[Ubuntu 17.4] Unable to locate package safeeyes

### ISSUE THREAD

**Issue by tranghaviet** [slgobinath/SafeEyes#228]
I can't install SafeEyes on ubunu 17 machine when run following commands:
```
sudo add-apt-repository ppa:slgobinath/safeeyes
sudo apt update
sudo apt install safeeyes
```
Please help me.

**Comment by slgobinath** [comment]
Ubuntu Launchpad kept Safe Eyes build in the waiting queue for 6 days. That was the reason for this issue. Even on my computer, I am using Safe Eyes installed using `pip`.
Now I have canceled the previous build and pushed a new version (2.0.2-3). Will see what happen...

**Comment by slgobinath** [comment]
Fixed with 2.0.2-3

**Comment by tranghaviet** [comment]
It worked. Thanks @slgobinath.
